src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -92,22 +94,65 @@ class WorkflowManagerActor(backend: Backend) extends Actor with CromwellActor {
     restartIncompleteWorkflows()
   }
 
+  private def reply[A](id: WorkflowId, fa: Future[A],
+                       onSuccess: (WorkflowId, A) => WorkflowManagerSuccessResponse,
+                       onFailure: (WorkflowId, Throwable) => WorkflowManagerFailureResponse): Unit = {
+    // Don't close over sender.
```

Comment:
I'd prefer making sure people understood over having that comment everywhere. Too bad you can't extract common comments into a constant or something ;)